src/ts/waterfall/details-overlay/html-details-body.ts

```diff
@@ -129,7 +138,7 @@ export function createDetailsBody(requestID: number, entry: WaterfallEntry, acco
       </div>
       ${timingsTab}
       <div class="tab raw-data">
-        <pre><code>${JSON.stringify(entry.rawResource, null, 2)}</code></pre>
+        <pre><code>${escapeHtml(JSON.stringify(entry.rawResource, null, 2))}</code></pre>
```

Comment:
can't request/response headers also contain single/double-quotes and should be escaped?